<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Object_Detection</class>
 <widget class="QMainWindow" name="Object_Detection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Detection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 250, 239);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>981</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Home</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Klasifikasi Aksara</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>AR Jawa Learn</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>241</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Setting</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>221</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Show Location Predict :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Show</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>SHOW LABEL :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Show</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>223</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="0">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="value_green">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>RED  :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="slider_green">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="value_red">
          <property name="minimumSize">
           <size>
            <width>37</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>3</width>
            <height>3</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="value_blue">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="slider_red">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="slider_blue">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>GREEN :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>BLUE :</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="3">
         <widget class="QPushButton" name="btn_reset_rgb">
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>RESET</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>661</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 247, 247);</string>
      </property>
      <property name="title">
       <string>MAIN WINDOW</string>
      </property>
      <widget class="QLabel" name="disp_main">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>640</width>
         <height>480</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 247, 247);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>0</y>
        <width>251</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 247, 247);</string>
      </property>
      <property name="title">
       <string>LOG</string>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>561</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 247, 247);
border-radius:3px;</string>
       </property>
      </widget>
     </widget>
     <widget class="QLCDNumber" name="lcd_clock">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 247, 247);</string>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 247, 247);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>241</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="1">
         <widget class="QLabel" name="Qlabel_cpu">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>CPU</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>FPS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Qlabel_fps">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(237, 85, 59);</string>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>RAM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="Qlabel_ram">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 247, 247);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="camlist">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>133</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>133</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_start">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>10</y>
         <width>100</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stop">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>100</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>About Me</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_red</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_blue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_green</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
